com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -84,7 +84,7 @@ Shader "Universal Render Pipeline/Lit"
             Cull[_Cull]
 
             HLSLPROGRAM
-            #pragma exclude_renderers d3d11_9x gles
+            #pragma exclude_renderers gles gles3 glcore
```

Comment:
Why glcore? is that correct?

Reply:
Hybrid doesn't support OpenGL or GLES so we exclude those and have the second subshader only for them.